src/email-helpers.js

```diff
@@ -22,11 +22,12 @@ try {
   // Do not set AWS_SDK_LOAD_CONFIG if aws config file is missing.
 }
 
-export async function sendEmail(filename, format, sender, recipient, transport, smtphost, smtpport, smtpsecure, smtpusername, smtppassword, subject) {
+export async function sendEmail(filename, url, sender, recipient, transport, smtphost, smtpport, smtpsecure, smtpusername, smtppassword, subject, note) {
```

Comment:
Minor: I feel the param list is a little bit long, we may consider to refactor it a little bit later.

Reply:
created issue #12 to track this